.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
+sudo: false
+cache:
+  directories:
+    - node_modules
```

Comment:
yeah that's probably something we don't want to do then as it will likely just bite us